services/spd/trusty/trusty.c

```diff
@@ -414,6 +414,14 @@ static int32_t trusty_setup(void)
 					    DAIF_IRQ_BIT |
 					    DAIF_ABT_BIT);
 
+	/*
+	 * arg0 = TZDRAM aperture available for BL32
+	 * arg1 = BL32 boot params
+	 * arg2 = BL32 boot params length
+	 */
+	ep_info->args.arg1 = ep_info->args.arg2;
+	ep_info->args.arg2 = 4096*2;
+
```

Comment:
This change doesn't seem to match the commit message.
Also, would it be better to have a constant for the params length?